wooConfig.example.js

```diff
@@ -0,0 +1,6 @@
+const wooConfig = {
+	siteUrl: 'https://xyz.com',
+	graphqlUrl: 'https://xxwordpresssitexx.com/graphql',
```

Comment:
We should move the config into a `.env` file
https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
@imranhsayed We already have `.env` file with the following content:
```
SITE_URL=http://localhost:3000
WOO_SITE_URL=http://yourwocommercesite.com
WOO_CONSUMER_KEY=xxxxx
WOO_SECRET=xxxxx

```

in that regard, may be we don't need this file `wooConfig.example.js`

Reply:
A few env variables are need server side while others client side. 
The one's we need client side must be prefixed by `NEXT_PUBLIC_`
e.g. NEXT_PUBLIC_WORDPRESS_SITE_URL=abcdefghijk
One's prefixed will be available both server side and client side.
We need WordPress site url for both client side and server side

We can remove :
```shell
WOO_CONSUMER_KEY=xxxxx
WOO_SECRET=xxxxx
```

Once we replace all instances we can safely remove this file and put `.env-example` instead